src/node.cc

```diff
@@ -759,9 +764,9 @@ Local<Value> ErrnoException(Isolate* isolate,
         String::Concat(cons3, String::NewFromUtf8(env->isolate(), path));
     Local<String> cons5 =
         String::Concat(cons4, FIXED_ONE_BYTE_STRING(env->isolate(), "'"));
-    e = Exception::Error(cons5);
+    e = v8::Exception::Error(cons5);
```

Comment:
I'm just thinking from the point of getting the changes into the community. This is one that might be a harder sell.  But I guess there is a change will have a fix on the compiler side by the time we need it to go in.